### PR TITLE
Add support for DurationBeforeTrigger

### PR DESCRIPTION
**Pull request by EtienneM**
Related to Scalingo/scalingo-alerter-service#93

### CODE REVIEW

File: alerts.go [hunk 1]
```diff
@@ -16,95 +16,115 @@ type AlertsService interface {
 
 var _ AlertsService = (*Client)(nil)
 
-type Alert struct {
+// AlertInternal is an alert with the time.Duration attributes being the string
+// representation. This is how the alerter service expects them.
+type AlertInternal struct {
```
Comment (Soulou) [inline alerts.go]: Isn't it possible to update the input of the alerting API, to standardize around ints?

Comment (EtienneM) [reply alerts.go]: Yes we probably could do that. You're right it will probably save us some headaches

File: alerts.go [hunk 2]
```diff
@@ -17,94 +17,100 @@ type AlertsService interface {
...
+	RemindEvery           string        `json:"remind_every"`
 }
 
 type AlertsRes struct {
 	Alerts []*Alert `json:"alerts"`
 }
 
 type AlertRes struct {
-	Alert Alert `json:"alert"`
+	Alert *Alert `json:"alert"`
```
Comment (Soulou) [inline alerts.go]: Why a pointer?

Comment (EtienneM) [reply alerts.go]: All methods of the `AlertsService` interface return a `*Alert`. So it's kind of straightforward for me to have a `*Alert` here.

Comment (Soulou) [reply alerts.go]: Ok, I'll open a issue saying not to use pointer then :D

**Pull request merged**